### ISSUE TITLE
Sooty Bee sorted incorrectly in Beepedia

### ISSUE THREAD

**Issue by g0urra** [NillerMedDild/Enigmatica6#3785]
### Modpack Version

0.5.20

### Describe your issue.

Sooty Bee seem to be sorted as Coal Bee, as evident in the alphabetical order in the Beepedia.

![bild](https://user-images.githubusercontent.com/12349921/143591926-086ebaae-9d24-493d-a648-67e6386b0964.png)

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Server

**Comment by TemperedFool** [comment]
Yeah, this one I'm aware of; the bees are sorted on ID rather than lang name, and the devs don't want to change that. Next time we have a world-breaking change I'm going to change the ID's of a bunch of bees (sooty included) to match their names, but don't want to do so currently as it will cause people to lose their bees.
